infra/terraform/test-org/org/locals.tf

```diff
@@ -20,14 +20,17 @@ locals {
   cft_ci_group     = "cft-ci-robots@test.infra.cft.tips"
   cft_dev_group    = "cft-developers@dev.infra.cft.tips"
   gcp_admins_group = "gcp-admins@test.infra.cft.tips"
+  tgm_org_name     = "terraform-google-modules"
```

Comment:
It would be clearer to hard code these values directly.

Reply:
done